### PR TITLE
Api 7534 saml proxy reorg

### PR DESCRIPTION
**Pull request by crolarlibertyva**
lighthouse-saml-proxy reorganized as a distinct project in its own git repo

### CODE REVIEW

**Comment by beaugrantham-va** [comment]
Are there other CICD changes that need to happen outside of this? I recall some work that had to be done for oauth-proxy. If so, can you link the stories/PRs?

File: saml-proxy/test/test-private-key.pem [hunk 1]
```diff
@@ -1,27 +0,0 @@
------BEGIN RSA PRIVATE KEY-----
```
Comment (beaugrantham-va) [inline saml-proxy/test/test-private-key.pem:1]: Are these certs not needed for the unit or regression tests?

Comment (crolarlibertyva) [reply saml-proxy/test/test-private-key.pem:1]: yes they are.

**Comment by crolarlibertyva** [comment]
> Are there other CICD changes that need to happen outside of this? I recall some work that had to be done for oauth-proxy. If so, can you link the stories/PRs?

 - https://github.com/department-of-veterans-affairs/devops/pull/9452

Tom temporarily applied the change so I can test out cicd for the repo.

**Review by lukemarkwordtlibertyits — APPROVED** [review]
LGTM

**Pull request merged**